hammock-asynchttpclient/src/main/scala/hammock/asynchttpclient/AsyncHttpClientInterpreter.scala

```diff
@@ -14,19 +14,16 @@ object AsyncHttpClientInterpreter {
 
   def apply[F[_]](implicit F: InterpTrans[F]): InterpTrans[F] = F
 
-  implicit def instance[F[_]: Async](
+  implicit def instance[F[_]: Sync](
       implicit client: AsyncHttpClient = new DefaultAsyncHttpClient()
   ): InterpTrans[F] = new InterpTrans[F] {
     override def trans: HttpF ~> F = transK andThen λ[Kleisli[F, AsyncHttpClient, *] ~> F](_.run(client))
   }
 
-  def transK[F[_]: Async]: HttpF ~> Kleisli[F, AsyncHttpClient, *] = {
+  def transK[F[_]: Sync]: HttpF ~> Kleisli[F, AsyncHttpClient, *] = {
 
     def toF[A](future: jc.Future[A]): F[A] =
-      Async[F].async(_(Try(future.get) match {
-        case Failure(err) => Left(err)
-        case Success(a)   => Right(a)
-      }))
+      Sync[F].blocking(future.get)
```

Comment:
Previously this was blocking a compute thread (as far as I can see) - the callback was invoked immediately and then blocked on the future completing. Either way, the interop with `jc.Future` isn't asynchronous.